### PR TITLE
added custom markers

### PR DESCRIPTION
**Pull request by MagdaN**

### CODE REVIEW

File: src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/Mapping/Mapping.ts [hunk 1]
```diff
@@ -74,12 +85,16 @@ export var mapInput = (
...
+            var selectedItemLeafletIcon = (<any>leaflet).icon(selectedItemIcon);
+
             // FIXME: Definetely Typed
             var marker = (<any>leaflet).marker();
+            marker.setIcon(selectedItemLeafletIcon);
 
             if (typeof scope.lat !== "undefined" && typeof scope.lng !== "undefined") {
                 marker.setLatLng(leaflet.latLng(scope.lat, scope.lng)).addTo(map);
                 marker.dragging.enable();
+
```
Comment (xi) [inline src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/Mapping/Mapping.ts]: whitespace

**Comment by xi** [comment]
I think @local-girl already worked on using an icon font for the markers. I would like to ask here before merging this.

**Comment by local-girl** [comment]
Markers are now font icons instead of SVGs

**Comment by MagdaN** [comment]
Looks good

**Pull request merged**